app.py

```diff
@@ -0,0 +1,54 @@
+import preprocessor as pp
+import pandas as pd
+import pickle
+from sklearn.feature_extraction.text import CountVectorizer
+
+class Application:
+    def __init__(self):
+        self.source_data = []
+        self.import_model_vocabulary()
+        self.preprocessor = pp.Preprocessor()
+        self.vectorizer = CountVectorizer(vocabulary=self.vocabulary)
+
+    def import_model_vocabulary(self):
```

Comment:
I would split this into 2 separate methods.

Reply:
Good idea.